src/strings/en-US/PowerFxResources.resw

```diff
@@ -2359,6 +2359,7 @@
   </data>
   <data name="AboutIndex_index" xml:space="preserve">
     <value>An integer index.</value>
```

Comment:
We can have a better description, something along the lines of `The index of the record to be retrieved`.

Reply:
nit:   (call out 1-based) "The index of the record to be retrieved, starting at 1"